public/app/plugins/datasource/loki/queryHints.ts

```diff
@@ -82,6 +85,7 @@ export function getQueryHints(query: string, series: DataFrame[]): QueryHint[] {
           label: `If ${levelLikeLabel} label has level values, consider using label_format to rename it to "level". Level label can be then visualized in log volumes.`,
           action: {
             type: 'ADD_LEVEL_LABEL_FORMAT',
+            hint: 'hint: add label level format',
```

Comment:
`rename level-like label to level to see levels in log volumes` ? We need help from technical writers, because naming this one is not easy. :D